HomeWork1/task16.py

```diff
@@ -0,0 +1,24 @@
+# Задача № 16 
+# Требуется вычислить, сколько раз встречается некоторое 
+# число X в массиве A. Пользователь в первой строке вводит 
+# натуральное число N – количество элементов в массиве. 
+# В последующих  строках записаны N целых чисел Ai (можно 
+# использовать псевдогенерацию). Последняя строка содержит число X.
+# *Пример:*
+#     5
+#     7 -2 3 5 1
+#     3
+#     -> 1
+
+import random
+n = int(input('Введите размерность массив N: '))
+lst = [random.randint(-5,5) for i in range (n)]
+print(lst)
+count = 0
+x = int(input('Введите искомое число Х: '))
+for item in lst:
+     if item == x in lst:
```

Comment:
Добавлять `in lst` не нужно
```suggestion
     if item == x:
```